mapie/classification.py

```diff
@@ -837,39 +832,42 @@ def predict(
             dtype=["float64", "object"]
         )
 
-        # Estimate probabilities from estimator(s)
-        y_pred = self.single_estimator_.predict(X)
-        if cv == "prefit":
-            y_pred_proba = self.single_estimator_.predict_proba(X)
-            y_pred_proba = self._check_proba_normalized(y_pred_proba)
-        else:
-            y_pred_proba_k = np.asarray(
-                Parallel(
-                    n_jobs=self.n_jobs, verbose=self.verbose
-                )(
-                    delayed(self._predict_oof_model)(estimator, X)
-                    for estimator in self.estimators_
-                )
-            )
-            if agg_scores == "crossval":
-                y_pred_proba = y_pred_proba_k[self.k_]
-                y_pred_proba = self._check_proba_normalized(
-                    y_pred_proba, axis=2
-                )
-            elif agg_scores == "mean":
-                y_pred_proba = np.mean(y_pred_proba_k, axis=0)
-                y_pred_proba = self._check_proba_normalized(
-                    y_pred_proba, axis=1
-                )
-            else:
-                raise ValueError("Invalid 'agg_scores' argument.")
-
         # Estimate prediction sets
+        y_pred = self.single_estimator_.predict(X)
         n = self.n_samples_val_
         if alpha_ is None:
             return np.array(y_pred)
 
         else:
+            # Estimate of probabilities from estimator(s)
+            # In all cases : len(y_pred_proba.shape) == 3
+            # with  (n_test, n_classes, n_alpha/n_conf_scores)
```

Comment:
The slash here is misleading, prefer "or"

Reply:
Done